source/extensions/filters/network/rocketmq_proxy/protocol.cc

```diff
@@ -735,9 +734,9 @@ void MetadataHelper::parseRequest(RemotingCommandPtr& request, MessageMetadataSh
   metadata->headers().addCopy(Http::LowerCaseString("code"), code);
 
   if (enumToInt(RequestCode::AckMessage) == code) {
-    metadata->headers().addCopy(Http::LowerCaseString(RocketmqConstants::get().BrokerName),
+    metadata->headers().addCopy(Http::LowerCaseString("broker_name"),
```

Comment:
```suggestion
    metadata->headers().addCopy(Http::LowerCaseString(BrokerName),
```
and same for other places. Make sure you have latest main branch which has this function

Reply:
Sorry for mistake, done.